bucket/neovim-gtk.json

```
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://github.com/daa84/neovim-gtk",
+    "version": "0.1.1_545",
+    "description": "GTK ui for neovim written in rust using gtk-rs bindings. With ligatures support.",
+    "license": {
+        "identifier": "GPL3",
+        "url": "https://github.com/daa84/neovim-gtk/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://ci.appveyor.com/api/buildjobs/lj7awg3qbi99hjp4/artifacts/nvim-gtk-mingw64.7z",
+            "hash": "E7DEF6E21C48AF2DD613081B69B49318C41112CD036AA33B465B0D2F7571BE60"
+        }
+    },
+    "extract_to": "neovim-gtk",
+    "bin": [
+        "neovim-gtk\\dist\\bin\\nvim-gtk.exe"
+    ],
+    "shortcuts": [
+        [
+            "neovim-gtk\\dist\\bin\\nvim-gtk.exe",
+            "Neovim GTK"
+        ]
+    ],
+    "suggest": {
+        "neovim": "extras/neovim"
```

```suggestion
        "neovim": "neovim"
```
Neovim is in the main bucket: https://github.com/ScoopInstaller/Main/blob/master/bucket/neovim.json